package.json

```diff
@@ -22,6 +22,7 @@
     "@tanstack/react-query-devtools": "^5.37.1",
     "axios": "^1.4.0",
     "dotenv": "^16.0.3",
+    "esbuild": "^0.21.4",
```

Comment:
What is this package? @mayura-andrew

Reply:
i tried to test to run npm run build in my local but i got  that the package "@esbuild/linux-x64" could not be found, which is needed by esbuild.  then i install ```npm install esbuild```.  i